.github/scripts/repair-github-releases.mjs

```
@@ -0,0 +1,161 @@
+        fs.writeFileSync(tmp, section + '\n');
+        run('gh', [
+          'release',
+          'create',
+          tag,
+          '--target',
+          sha,
+          '--notes-file',
+          tmp,
+        ]);
+      } finally {
+        try { fs.unlinkSync(tmp); } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)